Final pull request

philosopherdog opened the pull request:
Did some final tweaks this morning.

philosopherdog committed:
Track 1 files into repository.

- untracked images/github-200.png

Auto commit by GitBook Editor

philosopherdog committed:
Updates README.md

Cleans up wording
Removes poorly worded min sys requirements
Removes setup material that is repeated in later sections

philosopherdog committed:
Updates section0/0.2.md

fixes outdated xcode setup instructions

philosopherdog committed:
Updates section0/0.2.md

Fixes the instructions that say to drag your xcode from downloads to the dock! Adds a 6 sec video instead of the screenshot

philosopherdog committed:
Track 1 files into repository.

- untracked images/github-add-remote.png

Auto commit by GitBook Editor

philosopherdog committed:
Updates section0/setup-terminal.md

Adds setup github section
Adds setup Terminal section

philosopherdog committed:
Updates section0/0.3.md

Removes useless git resources and adds useful ones.

philosopherdog committed:
Updates section0/0.3.md

Fixes up the git/github instructions including some reformatting fixes.

philosopherdog committed:
Updates section0/0.3.md

Amends last commit with different emoji.

philosopherdog committed:
Updates section2/README.md

Edits resources session in Introduction to Objc by removing irrelevant resources and adding an excellent video that explains what OO is in a concise way.

philosopherdog committed:
Updates section2/2.1.md

Edits the "Your First App section

philosopherdog committed:
Updates section2/2.2.md

Removed an irrelevant resource called the fundamentals of tableviews, which was just a listing of apple's sample code.

philosopherdog committed:
Updates section3/3.1.md

Removed the separate git/github section and moved the instructions to push to git inside the objective-c section. The intro section was just repetition of early stuff and one page doesn't warrant a whole new section.

philosopherdog committed:
Updates README.md

Removed references to the separate git section mentioned in the intro. Also removed gitmagic from the git resources since I will add them to the end of the document.

philosopherdog committed:
Cleans up some of the mess the editor left move things around.

philosopherdog committed:
adds a couple of new files to organize the resources

philosopherdog committed:
Restructures sections and renumbers stuff.

philosopherdog committed:
Separates TODO app assignment from Objective-c
Adds the stubs for the main Objc resources I will put down, one link for each of 3 resources.
Moves the assignments for objc into the end of the objc intro section
Renames the objc assignments so that assignments are numbered consistently!

philosopherdog committed:
Adds a new C video tutorial series.

philosopherdog committed:
Refactors the push to git section.

philosopherdog committed:
Reorders the Intro to Objective C sestion to make the video the first resource.

philosopherdog committed:
Adds intro to Objc and explains why we start with it.
Adds some blank files in preparation for the other resources in this section.

philosopherdog committed:
Adds Lynda.com resource to Objc intro section.

philosopherdog committed:
Adds Code School resource to Intro to Objc section.

philosopherdog committed:
Adds BNR resource.

philosopherdog committed:
Adds nice-to-have and stretch resources.

philosopherdog committed:
Adds Git Magic to the nice-to-have resources

philosopherdog committed:
Adds section on Where to get the BNR for free/paid. Some rewording on the Additional Resources section.

philosopherdog committed:
Adds missing links to the code school resource.

philosopherdog committed:
Changes the numbering of the objc assignments to be more logical.

philosopherdog committed:
adds missing "you"

philosopherdog committed:
Could of grammar changes

philosopherdog committed:
Small grammar change

philosopherdog committed:
Small grammar error.

philosopherdog committed:
commands below

philosopherdog committed:
missing word

philosopherdog committed:
Minor grammar changes

philosopherdog committed:
minor grammar correction

philosopherdog committed:
Updates section4/4.1.md

Updates to Liam's suggestions.

philosopherdog committed:
Updates section3/strech.md

Final edit pass.

philosopherdog committed:
Adds Change log with main changes documented.

philosopherdog committed:
minor tweaks after reviewing the diffs on github

review-ninja commented:
<a href="https://app.review.ninja/lighthouse-labs/ios-prep-book/pull/24" target="_blank"><img src="https://app.review.ninja/49509218/pull/24/badge" alt="ReviewNinja"/>

kvirani committed:
Minor improvements

- Fix typo about pluralization. Toyota’s => Toyotas
- Changelog mentions section numbers
- Other minor formatting fixes
